projects/documentation/src/components/layout.ts

```diff
@@ -69,11 +70,11 @@ const DEFAULT_SCALE = (
         ? localStorage.getItem(SWC_THEME_SCALE_KEY) || SCALE_FALLBACK
         : SCALE_FALLBACK
 ) as Scale;
-const DEFAULT_THEME = (
+const DEFAULT_SYSTEM = (
     window.localStorage
-        ? localStorage.getItem(SWC_THEME_THEME_KEY) || THEME_FALLBACK
-        : THEME_FALLBACK
-) as ThemeVariant;
+        ? localStorage.getItem(SWC_THEME_THEME_KEY) || SYSTEM_FALLBACK
```

Comment:
This key doesn't match the set key below and will prevent persistence of system choice.

We should also use a conversion period where in we read either of the keys to ensure that choice is persisted across this change.

Reply:
I have created a `DEFAULT_THEME_CHOICE` constant to read the theme or system of choice from the localStorage
```
const DEFAULT_THEME_CHOICE = (
    window.localStorage
        ? localStorage.getItem(SWC_THEME_THEME_KEY) ||
          localStorage.getItem(SWC_THEME_SYSTEM_KEY) ||
          SYSTEM_FALLBACK
        : SYSTEM_FALLBACK
) as SystemVariant;
```